src/metabase/query_processor/middleware/fetch_source_query.clj

```diff
@@ -152,7 +144,7 @@
        (log/info (trs "Found substitute cached query for card {0} from {1}.{2}"
                       card-id
                       (ddl.i/schema-name {:id database-id} (public-settings/site-uuid))
-                      (:table_name card))))
+                      (:table_name persisted-info))))
```

Comment:
We used to select the card and persisted info into one object. Now two queries but never updated where we got `:table_name`. So we were logging "found substitute cached query for card 1234 from cache_schema.null"